lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -100,10 +106,27 @@ public void setInstalledFeatureList(List<Feature> installedFeatureList) {
         this.installedFeatureList = installedFeatureList;
     }
 
-    public void initConfigFileList() {
+    public String getContainerName() {
+        return containerName;
+    }
+
+    public void setContainerName(String containerName) {
+        this.containerName = containerName;
+    }
+
+    public boolean hasRunningContainer() {
+        return this.runningContainer;
+    }
+
+    public void setRunningContainer(boolean container) {
```

Comment:
The variable name is odd. Makes me think it identifies the container some how. But this boolean is indicating whether the container is alive or not, right?

Reply:
Correct, but I'll rename it for better clarity